contrib/builder/deb/ubuntu-debootstrap-vivid/Dockerfile

```diff
@@ -0,0 +1,12 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/deb/generate.sh"!
+#
+
+FROM ubuntu-debootstrap:vivid
```

Comment:
You're going to want to swap these to just use the `ubuntu` image, like we did in Docker.